invoices/resolution_result.go

```diff
@@ -142,7 +142,7 @@ func (f FailResolutionResult) FailureString() string {
 		return "invoice already canceled"
 
 	case ResultInvoiceAlreadySettled:
-		return "invoice alread settled"
+		return "invoice already settled"
```

Comment:
I hope nobody is string matching against this error in the wild yet... Could very well be though, since we don't have nice error codes for things. Not sure if it's worth potentially breaking wallets. I did do a quick google search with this exact string and didn't find anything...

Reply:
@yyforyongyu what do you think? This might be an error that apps already string match against...

Reply:
yeah i guess that's just the pain we must suffer...a typo in the error message is quite ugly🙈